docs/installation-and-operations/changing-to-bim-edition/README.md

```diff
@@ -77,3 +77,33 @@ OpenProject installations with a valid Enterprise Edition token can switch to th
 theme.
 
 Go to _-> Administration -> Design_ and from the _Themes_ drop down menu choose _OpenProject BIM_.
+
+### Docker
+
+##### OpenProject BIM Edition with Docker Compose
+
+If you would like to use OpenProject BIM Edition inside your docker-compose containers, in your `docker-compose.yml` file in `x-op-app` > `environment` add the line
+
+```yml
+    OPENPROJECT_EDITION: "bim"
+```
+
+This could look like this after you edited the file:
+
+```yml
+x-op-app: &app
+  <<: *image
+  <<: *restart_policy
+  environment:
+    RAILS_CACHE_STORE: "memcache"
+    OPENPROJECT_CACHE__MEMCACHE__SERVER: "cache:11211"
+    OPENPROJECT_RAILS__RELATIVE__URL__ROOT: "${OPENPROJECT_RAILS__RELATIVE__URL__ROOT:-}"
+    OPENPROJECT_EDITION: "bim"
+    DATABASE_URL: "postgres://postgres:p4ssw0rd@db/openproject"
+    USE_PUMA: "true"
+    # set to true to enable the email receiving feature. See ./docker/cron for more options
+    IMAP_ENABLED: "${IMAP_ENABLED:-false}"
+```
+
+Note: If your current Docker installation does not yet hold important information I recommend you to simply create all docker containers from scratch as the seeded data such as themes, types, and demo projects are different in the BIM edition. The seed data gets seeded only the very first time you call.
```

Comment:
Do we write from the the first person's perspective? Looks weard to me to see "I recommend" in a guide.

Reply:
Hi @wielinde 
actually i might have found an answer to your question.
There is a Style Guide that I will need to read, no one requested that yet, so i did not know:
Thanks for asking:
https://www.openproject.org/docs/development/contribution-documentation/documentation-style-guide/

Reply:
@adam-op I would also prefer another way to say this. I don't think the style guide says anything about this yet but still.

Suggestion: "[...] important information, it would be easiest to re-create all containers from scratch"

You may also need to clarify that starting from scratch will also require to actually delete the volumes (`docker-compose down --volumes`).

Reply:
- First person will be replaced by e.g. usage of passive.
- deleting the volumes will be added as suggested